### PR TITLE
Draft : [ASP.NET Core] Add http.route to http.client.duration

### PR DESCRIPTION
**Pull request by EnDimeonRus**
Fixes #.

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] Appropriate `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: src/OpenTelemetry/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@
   `AddEnvironmentVariableDetector` extension (Logs)
   ([#3889](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3889))
 
+* **Breaking change** Added `http.route` tag in `http.client.duration` metric
```
Comment (EnDimeonRus) [inline src/OpenTelemetry/CHANGELOG.md:10]: Maybe not so breaking

**Comment by vishweshbankwar** [comment]
@EnDimeonRus FYI - `http.route` is not part of client metric dimensions as per spec. See https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#attributes

**Comment by EnDimeonRus** [comment]
> @EnDimeonRus FYI - `http.route` is not part of client metric dimensions as per spec. See https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#attributes

Thansk for clarification. Looks like tag AttributeNetPeerName is what we need

**Comment by EnDimeonRus** [comment]
PR closed as not needed

**Pull request closed without merging**